lib/rubocop/cop/sorbet/sigils/enforce_sigil_order.rb

```diff
@@ -99,6 +108,30 @@ def check_magic_comments_order(tokens)
             end
           end
         end
+
+        def each_extra_empty_line(lines)
+          prev_line = 1
+
+          lines.each do |cur_line|
+            if (cur_line - prev_line) > 0
+              # we need to be wary of comments since they
+              # don't show up in the tokens
+              ((prev_line + 1)...cur_line).each do |line|
+                next unless processed_source[line-1].empty?
+
+                yield source_range(processed_source.buffer, line, 0)
+              end
+            end
+
+            prev_line = cur_line
+          end
+        end
+
+        private
+
+        def previous_and_current_lines_empty?(line)
+          processed_source[line - 2].empty? && processed_source[line - 1].empty?
+        end
```

Comment:
These 2 helpers methods were taken from https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/cop/layout/empty_lines.rb and adapted for our use case.